main.py

```diff
@@ -419,23 +431,30 @@ def gcs_to_pubsub(cloud_event: CloudEvent):
     Returns:
         None
     """
-    logging.info("Cloud event triggered")
+    logging.debug("Cloud event triggered")
 
     data = cloud_event.data
+
+    # Skip processing if filename starts with "temp"
+    if "/temp_" in data["name"]:
+        logging.warning("Blob name contains '/temp_', skipping processing")
+        return
+
+    # Skip processing folders
+    if data["name"][-1] == "/":
+        logging.warning("Blob name ends in '/', skipping processing")
```

Comment:
is this an unexpected thing to encounter? If so, logging.warning is fine. Otherwise lower it to info